sounds/views.py

```diff
@@ -297,7 +298,26 @@ def pack_download(request, username, pack_id):
     Download.objects.get_or_create(user=request.user, pack=pack)
     pack_sounds = pack.sound_set.filter(processing_state="OK",
             moderation_state="OK").select_related('user', 'license')
-    return download_sounds(pack_sounds, reverse('pack', args=[username, pack.id]))
+    licenses_url = (reverse('pack-licenses', args=[username, pack_id]))
+    sounds_url = reverse('pack', args=[username, pack_id])
+    return download_sounds(pack_sounds, sounds_url, licenses_url)
+
+
+def pack_licenses(request, username, pack_id):
+    pack = get_object_or_404(Pack, id=pack_id)
+    sounds_list = pack.sound_set.filter(processing_state="OK",
+            moderation_state="OK").select_related('user', 'license')
+    users = User.objects.filter(sounds__in=sounds_list).distinct()
+    sounds_url = reverse('pack', args=[username, pack_id])
+    licenses = License.objects.all()
```

Comment:
This list of licenses should be only the licenses which are in sounds of this pack, not all available licenses